pod/live/views.py

```diff
@@ -96,7 +100,17 @@ def video_live(request, slug):  # affichage des directs
                 'heartbeat_delay': HEARTBEAT_DELAY
             }
         )
+    # Search if broadcaster is used to display a BBB streaming live
+    # for which students can send message from this live page
+    display_chat = False
+    if USE_BBB and USE_BBB_LIVE:
+        livestreams_list = Livestream.objects.\
+            filter(broadcaster_id=broadcaster.id)
+        if len(livestreams_list) > 0:
```

Comment:
Je remarque que tu vérifies souvent avant une boucle la taille de ta liste,
Je crois que la boucle for ne fait rien si l'itérable est vide

```Python
for n in []:
    print(n)
````
Out : 
```Python
 Nothing
```